### PR TITLE
Add README file with usage examples to the CLI module

### PR DESCRIPTION
**Pull request by b-abel**
This adds a README file to the module `cli` with some example commands. This makes it easier to find the right commands and parameters for receiving messages and sending commands, especially with the new variants for Kafka-based messaging.

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+
+### Examples with Kafka based messaging 
+
+Receive *events* and *telemetry data* sent to the configured Kafka cluster for a tenant:
+
+```bash 
+# in directory hono/cli/target/
+java -jar hono-cli-*-exec.jar --hono.kafka.commonClientConfig.bootstrap.servers=$KAFKA_SERVERS --hono.client.username=consumer@HONO --hono.client.password=verysecret --tenant.id=$MY_TENANT --spring.profiles.active=receiver,kafka
```
Comment (kaniyan) [inline cli/README.md:55]: The parameters `--hono.client.username=consumer@HONO` and `--hono.client.password=verysecret` are not needed here as they are only needed for AMQP based messaging.

File: cli/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+Send a *command* to a device over the configured Kafka cluster:
+
+```bash 
+# in directory hono/cli/target/
+java -jar hono-cli-*-exec.jar --hono.kafka.commonClientConfig.bootstrap.servers=$KAFKA_SERVERS --hono.client.username=consumer@HONO --hono.client.password=verysecret --tenant.id=$MY_TENANT --device.id=$MY_DEVICE --spring.profiles.active=command,kafka
```
Comment (kaniyan) [inline cli/README.md:62]: The parameters `--hono.client.username=consumer@HONO` and `--hono.client.password=verysecret` are not needed in Kafka as they are only needed for AMQP based messaging.

Comment (b-abel) [reply cli/README.md]: Good catch! Removed i.

**Review by kaniyan — APPROVED** [review]
LGTM

**Pull request merged**